src/views/List.vue

```diff
@@ -1,40 +1,54 @@
 <template>
   <div>
+    <br>
     <h1>List</h1>
     <p>{{list}}</p>
     <p>{{list.listItems}}</p>
-    <v-data-table
-      :headers="headers"
-      :items="list.listItems"
-      :items-per-page="5"
-      class="elevation-1"
-    ></v-data-table>
+    <ol style="list-style-type:none;">
+      <li v-for="(item,i) in listItems" :key="`${item.text}${i}`">
+        <list-item :listItem="item"/>
+      </li>
+    </ol>
   </div>
 </template>
 <script>
+import ListItem from '../components/ListItem.vue';
+
 export default {
   name: 'List',
   props: ['title'],
```

Comment:
You should use the more verbose declarations from here on out. It will keep your future self happy. (Consult the Vue documentation for this, if you don't know it.) You should supply all fields and ensure your description is verbose.

Reply:
I am not sure what I was asked to do. 
Is it about usage of @ in path of import ListItem?